src/Approximations/iterative_refinement.jl

```diff
@@ -97,30 +98,33 @@ the new approximation is returned by this function.
 """
 function addapproximation!(Ω::PolygonalOverapproximation,
                            p1::Vector{N}, d1::Vector{N}, p2::Vector{N},
-                           d2::Vector{N})::LocalApproximation{N} where {N<:Real}
+                           d2::Vector{N})::LocalApproximation{N} where N<:Real
 
     approx = new_approx(Ω.S, p1, d1, p2, d2)
     push!(Ω.approx_list, approx)
     return approx
 end
 
 """
-    refine(Ω::PolygonalOverapproximation, i::Int)::Tuple{LocalApproximation, LocalApproximation}
+    refine(Ω::PolygonalOverapproximation,
+           R::LocalApproximation
+          )::Tuple{LocalApproximation, LocalApproximation}
 
-Refine a given local approximation of the polygonal approximation of a convex set,
-by splitting along the normal direction to the approximation.
+Refine a given local approximation of the polygonal approximation of a convex
+set by splitting along the normal direction to the approximation.
 
 ### Input
 
-- `Ω`   -- polygonal overapproximation of a convex set
-- `i`   -- integer index for the local approximation to be refined
+- `Ω` -- polygonal overapproximation of a convex set
+- `i` -- integer index for the local approximation to be refined
 
 ### Output
 
 The tuple consisting of the refined right and left local approximations.
 """
-function refine(Ω::PolygonalOverapproximation, i::Int)::Tuple{LocalApproximation, LocalApproximation}
-    R = Ω.approx_list[i]
+function refine(Ω::PolygonalOverapproximation,
```

Comment:
moreover, `Ω` is not needed anymore, i suggest to remove it from the function's signature.

Reply:
false -- i forgot the lines just below.

Reply:
Actually, we only need `Ω.S`. I will pass that instead.